src/test/java/BQJDBC/QueryResultTest/PreparedStatementTests.java

```diff
@@ -140,6 +140,59 @@ public void ParameterlessTest() {
         } catch (SQLException e) {
             Assert.fail(e.toString());
         }
+        try {
+            con.close();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        con = null;
+    }
+
+    /**
+     * This test ensures that getColumnType supports the following types: DOUBLE, BOOLEAN, BIGINT, VARCHAR, TIMESTAMP, DATE
+     * Note: RECORD/STRUCT are not tested here because PreparedStatementsTests run queries on Legacy SQL
+     */
+    @Test
+    public void ResultSetMetadataFunctionTestTypes() {
+        final String[] queries = new String[]{
+                "SELECT 3.14",
+                "SELECT TRUE",
+                "SELECT 1",
+                "SELECT 'test'",
+                "SELECT CAST(CURRENT_TIMESTAMP() AS TIMESTAMP)",
+                "SELECT CAST(CURRENT_DATE() AS DATE)",
+        };
+        final int[] expectedType = new int[]{
+                java.sql.Types.DOUBLE,
+                java.sql.Types.BOOLEAN,
+                java.sql.Types.BIGINT,
+                java.sql.Types.VARCHAR,
+                java.sql.Types.TIMESTAMP,
+                java.sql.Types.DATE,
+        };
+
+        for (int i = 0; i < queries.length; i ++) {
+            System.out.println("Running query: " + queries[i]);
```

Comment:
If you'd like me to! There are a bunch of other tests in that file that `System.out.print` the queries they'll run.

Reply:
I figured I'd leave in the `println` just for easier debugging while running these tests, and the others follow the same pattern - would you prefer to avoid `print`s in the test files?

Reply:
Maybe just a personal preference. Typically, I try to avoid them.

Reply:
Fair enough. I'll go through and remove this and other print statements in this test file.

Reply:
👍